src/interface.ts

```
@@ -0,0 +1,22 @@
+import { ChangeEvent, MouseEvent } from "react";
+
+export interface XploresAIType {
+  handleInputChange: (e: ChangeEvent<HTMLTextAreaElement>) => void;
+  handleSubmit: (e: MouseEvent<HTMLButtonElement>) => Promise<void>;
+  handleFileUpload: (e: FileUploadEvent) => void;
+  value: string;
+  loading: Boolean;
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)